### ISSUE TITLE
Data repeated  on 3/11/20 and 3/12/20

### ISSUE THREAD

**Issue by selvaggi** [CSSEGISandData/COVID-19#804]
for example deaths in italy are the same ofr these two dates.

**Comment by JimBudde** [comment]
I agree, something wonky occurred in the 3/12/2020 data set.  Italy's number clearly duplicated from previous day.  Will continue to troubleshoot and provide any further info I can.

**Comment by jescuderoma** [comment]
This problem affects the data from 3-12-2020 of, at leat, Italy, Spain, France and Germany.

**Comment by showdownhero** [comment]
All the data in the Australian states was impacted by this too

**Comment by nagyben** [comment]
Same issue observed with UK

**Comment by lglip** [comment]
Duplicate of #619 (and others)

The author addresses the issue in #650

The issue is known but corrections have not been implemented yet. Hopefully JHU CSSE will commit corrections in the near future.

**Comment by Hikmer** [comment]
Somewhat unacceptable that this data is so off the mark.  People are in panic based on bad data.

**Comment by sgt101** [comment]
This is not fixed for me yet

**Comment by delcacho** [comment]
It happened again today.

**Comment by sgt101** [comment]
This is really worrying me now - I am seeing infographics with projections based on fit to the repeated data points circulating. I think that this could influence decision making and could be fixed easily. Why is this being ignored when I have put a pull request and I am sure others have? This is really poor.
